sw/airborne/firmwares/rotorcraft/stabilization/wls/qr_solve.c

```diff
@@ -0,0 +1,1616 @@
+/*
+ * This is part of the qr_solve library from John Burkardt.
```

Comment:
I am interested to try and compare, though I need to find time for that. 

At the time of coding we were not aware that this was already in paparazzi...